.github/workflows/ci.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
         pip list
     - name: Test with pytest
       run: |
-        python -m pytest -m "not atlas_xaod_runner and not cms_aod_runner"
+        python -m pytest -m "not atlas_xaod_runner and not cms_runner"
```

Comment:
I think this needs all three things on the command line now - to prevent any docker runnings at all.